### PR TITLE
Autoupdate configuration for UART.

### PR DESCRIPTION
**Pull request by microsoftsam**

### CODE REVIEW

File: pxtarget.json [hunk 1]
```diff
@@ -84,6 +84,16 @@
                     "microbit-pins": "",
                     "microbit-serial": ""
                 }
+            }, 
+            {
+                "type": "api",
+                "map": {
+                    "serial\\.writeNumber\\((.*?)\\)" : "bluetooth.uartWriteNumber($1)",
+                    "serial\\.writeString\\((.*?)\\)" : "bluetooth.uartWriteString($1)",
+                    "serial\\.writeValue\\((.*?)\\)" : "bluetooth.WriteValue($1)",
```
Comment (pelikhan) [inline pxtarget.json]: remove this line

File: pxtarget.json [hunk 2]
```diff
@@ -84,6 +84,16 @@
                     "microbit-pins": "",
                     "microbit-serial": ""
                 }
+            }, 
+            {
+                "type": "api",
+                "map": {
+                    "serial\\.writeNumber\\((.*?)\\)" : "bluetooth.uartWriteNumber($1)",
+                    "serial\\.writeString\\((.*?)\\)" : "bluetooth.uartWriteString($1)",
```
Comment (pelikhan) [inline pxtarget.json]: remove this line

File: pxtarget.json [hunk 3]
```diff
@@ -84,6 +84,16 @@
                     "microbit-pins": "",
                     "microbit-serial": ""
                 }
+            }, 
+            {
+                "type": "api",
+                "map": {
+                    "serial\\.writeNumber\\((.*?)\\)" : "bluetooth.uartWriteNumber($1)",
```
Comment (pelikhan) [inline pxtarget.json]: remove this line

File: pxtarget.json [hunk 4]
```diff
@@ -84,6 +84,16 @@
...
+                    "serial\\.writeString\\((.*?)\\)" : "bluetooth.uartWriteString($1)",
+                    "serial\\.writeValue\\((.*?)\\)" : "bluetooth.WriteValue($1)",
+                    "bluetooth\\.uartRead\\((.*?)\\)" : "bluetooth.uartReadUntil($1)",
+                    "bluetooth\\.uartWrite\\((.*?)\\)" : "bluetooth.uartWriteUntil($1)"
```
Comment (pelikhan) [inline pxtarget.json]: This change requires the UART PR to go in first.

Comment (microsoftsam) [inline pxtarget.json]: Yeah, hence the merge is into the UART PR branch.

**Pull request merged**